### PR TITLE
Iss1b

### PR DESCRIPTION
**Pull request by lilielkhalil**
Hello,

Kindly find my first try for git

### CODE REVIEW

**Comment by lilielkhalil** [comment]
Hello, Kindly find my article below

**Comment by lunebellec** [comment]
thanks @lilielkhalil could you please move this text in the README.md file? That's what is rendered as your project home page.

Also please add a few sentences on your research interests and your background.

File: article.md [hunk 1]
```diff
@@ -3,5 +3,24 @@ This is my first article
 
 Introduction
 
+Personal information
+
 My name is lili el khalil
-Master student at UdeM
+Born in 03 august 1998, my birthday is in 4 days !!
```
Comment (lunebellec) [inline article.md]: haha! Good to know!! I am going to let people know on discord. But I suggest to remove your exact birthday from this document. It is silly, but birthday is a security question in many services, so you should not make this information too easy to find on the web.

File: article.md [hunk 2]
```diff
@@ -3,5 +3,24 @@ This is my first article
...
+My status in Canada is as an International student
```
Comment (lunebellec) [inline article.md]: Once again, I suggest removing this type of personal information from a public document.

File: article.md [hunk 3]
```diff
@@ -3,5 +3,24 @@ This is my first article
...
+
+Educational backgroung
+
+Till present : i'm doing my masters in psychology at UdeM
+I Did my bachelor in clinical psychology in Lebanon and i wish to continue my doctorate in Canada also in clinical psychology
+
+Working Experience :
+
+I Can say that i totally work in a different field than my study !
+I do have more as administratif and finance experience.
```
Comment (lunebellec) [inline article.md]: suggestion: I had experience in administration and finance

File: README.md [hunk 4]
```diff
@@ -3,3 +3,24 @@
    <img src="https://avatars.githubusercontent.com/u/1670887?v=4?s=100" width="100px;" alt=""/>
```
Comment (lunebellec) [inline README.md:3]: you need to update this information

File: README.md [hunk 5]
```diff
@@ -3,3 +3,24 @@
...
    <br /><sub><b>Pierre bellec</b></sub>
 </a>
+
+Hello, This is my first article
```
Comment (lunebellec) [inline README.md:7]: consider removing. Also I commented on the wrong document by mistake. See my comment in the other file.

File: article.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+Hello,
```
Comment (lunebellec) [inline article.md]: please remove this file, which has duplicate information with the readme.

**Review by lunebellec — CHANGES_REQUESTED** [review]

**Comment by lilielkhalil** [comment]
Bonjour,

Les changements a été faite maintenant avec succés

Mercii
Lili
________________________________

**Comment by lunebellec** [comment]
It still looks like the avatar on this branch is my picture. 
But I see this was updated on the main branch.
So it should get automatically merged, let's try!

**Review by lunebellec — APPROVED** [review]

**Pull request merged**